packages/geoadmin-log/package.json

```diff
@@ -19,16 +17,10 @@
         "dev": "vite",
         "type-check": "vue-tsc -p tsconfig.json",
         "generate-types": "vue-tsc --declaration",
-        "build": "npm run type-check && npm run generate-types && vite build",
-        "build:dev": "npm run build -- --mode development",
-        "build:int": "npm run build -- --mode integration",
-        "build:prod": "npm run build -- --mode production"
-    },
-    "devDependencies": {
-        "vite": "^6.0.11",
-        "vite-plugin-dts": "^4.5.0",
-        "vitest": "^3.0.4",
-        "vue-tsc": "^2.2.0"
+        "build": "pnpm run type-check && pnpm run generate-types && vite build",
+        "build:dev": "pnpm run build -- --mode development",
+        "build:int": "pnpm run build -- --mode integration",
+        "build:prod": "pnpm run build -- --mode production"
     },
     "peerDependencies": {
         "proj4": "^2.15.0"
```

Comment:
can we also move that into the pnpm catalog?

Reply:
Yes indeed!